lib/tbot/renew.go

```diff
@@ -98,6 +98,8 @@ func (b *Bot) AuthenticatedUserClientFromIdentity(ctx context.Context, id *ident
 	}
 
 	c, err := authclient.Connect(ctx, authClientConfig)
+
+	// TODO: Ping server and if it does not indicate FIPS enabled, reject.
```

Comment:
Should tbot refuse to connect to an Auth Server which does not respond to the Ping RPC with `IsBoring: true` ?

Reply:
https://github.com/gravitational/teleport/pull/23563/commits/c210a8bf8cfe462f8bb467b2f16af403980a7fd4